### ISSUE TITLE
[ENTRY] Tackling quantum phase transitions and barren plateaus in VQE with tensor networks

### ISSUE THREAD

**Issue by echertkov** [XanaduAI/QHack2021#66]
### Team Name: 

Hooked on Photonics

### Project Description: 

One of the most promising applications of variational quantum algorithms is the study of condensed matter phenomena, such as quantum phase transitions, with near-term quantum computers. Yet a major challenge in the successful application of variational quantum algorithms is the barren plateau phenomenon, where gradients become exponentially small in the number of qubits [1]. While barren plateaus have been observed for certain types of variational quantum circuits and cost functions [2], it is unclear whether the phenomenon would significantly hinder the simulation of condensed matter systems. Our goal in this project is to explore this problem and develop strategies for avoiding barren plateaus in the study of quantum phase transitions.

In particular, we use the variational quantum eigensolver (VQE) [3] to find the ground states of the transverse field Ising model, a spin chain whose ground state is known to undergo a quantum phase transition. To avoid the barren plateau phenomenon in our analysis of this model, we train variational circuits with physically relevant structure, such as tree tensor networks (TTN) and the multi-scale entanglement renormalization ansatz (MERA) [4]. The MERA, a tensor network used to study quantum critical systems, is particularly well-suited to our task. We show that for our problem TTN’s and MERA’s generally produce larger gradients than a hardware-efficient ansatz (HEA) typically used in VQE and thereby are easier to train and help alleviate barren plateaus.

[1] McClean, J.R., Boixo, S., Smelyanskiy, V.N. et al. Barren plateaus in quantum neural network training landscapes. Nat Commun 9, 4812 (2018).
[2] M. Cerezo, Akira Sone, Tyler Volkoff, Lukasz Cincio, Patrick J. Coles. Cost-Function-Dependent Barren Plateaus in Shallow Quantum Neural Networks. arXiV:2001.00550 (2020).
[3] Peruzzo, A., McClean, J., Shadbolt, P. et al. A variational eigenvalue solver on a photonic quantum processor. Nat Commun 5, 4213 (2014).
[4] G. Evenbly and G. Vidal. Algorithms for entanglement renormalization. Phys. Rev. B 79, 144108 (2009).

### Presentation: 

https://docs.google.com/presentation/d/1LVuJPog0zjqgti-RxUhz_KTHlVRE9mvsXhADUpUo7Tc/edit?usp=sharing

### Source code: 

https://github.com/echertkov/qhack_vqe_ttn

**Comment by co9olguy** [comment]
Thanks for the submission! We hope you have enjoyed participating in QHack :smiley: 

We will be assessing the entries and contacting the winners separately. Winners will be publicly announced sometime in the next month.

We will also be freezing the GitHub repo as we sort through the submitted projects, so you will not be able to update this submission.
